move/operators/sources/operators.move

```diff
@@ -121,30 +122,55 @@ module operators::operators {
         });
     }
 
-    /// Allows an approved operator to borrow a capability by its ID.
-    public fun borrow_cap<T: key + store>(
-        self: &Operators,
+    /// Allows an approved operator to temporarily loan out a capability by its ID.
+    /// The loaned capability must be restored by the end of the transaction.
+    public fun loan_cap<T: key + store>(
+        self: &mut Operators,
         _operator_cap: &OperatorCap,
         cap_id: ID,
         ctx: &mut TxContext
-    ): &T {
+    ): (T, Borrow) {
         assert!(self.operators.contains(&ctx.sender()), EOperatorNotFound);
         assert!(self.caps.contains(cap_id), ECapNotFound);
 
-        &self.caps[cap_id]
+        // Remove the capability from the `Operators` struct to loan it out
+        let cap = self.caps.remove<ID, T>(cap_id);
+
+        // Create a new `Referent` to store the loaned capability
+        let mut referent = borrow::new<T>(cap, ctx);
+
+        // Borrow the T capability and a Borrow hot potato object from the `Referent`
+        let (loaned_cap, borrow_obj) = borrow::borrow<T>(&mut referent);
```

Comment:
```suggestion
        let (loaned_cap, borrow_obj) = borrow::borrow(&mut referent);
```
Is the T qualifier needed or is it inferred automatically?

Reply:
automatically